### PR TITLE
Add mobile banking SCB support

### PR DESCRIPTION
**Pull request by tookaa2**
## Purpose

Add Mobile Banking payment option and SCB Mobile Banking support

## Description

- Added Mobile Banking SCB support to payment methods list (in example app capability toggles and actual payment selection)

## Quality assurance

- Mobile Banking SCB option available in capabilities settings (SCB - Mobile )
Screen Shot 2563-10-15 at 15 20 08
- New payment method added : Mobile Banking
Screen Shot 2563-10-15 at 15 20 20
- Mobile Banking SCB added to payment list.
Screen Shot 2563-10-15 at 15 20 29

- Mobile Banking SCB can be selected and sources can be created. (currently must use staging live dashboard credentials)
1. Make sure to enable Mobile Banking SCB in your account
1. Change PUBLIC_KEY in the project

## Operations impact

N/A

## Pre- and post-deployment steps

N/A

## Business impact (release notes)

N/A

## Add labels and assign reviewers

N/A

## Back-out Procedure

N/A

### CODE REVIEW

File: sdk/src/main/java/co/omise/android/models/SourceType.kt [hunk 1]
```diff
@@ -34,6 +34,11 @@ sealed class SourceType(
         data class Unknown(@JsonValue override val name: String?) : InternetBanking(name)
     }
 
+    sealed class MobileBanking(@JsonValue override val name: String?) : SourceType(name) {
+        object Scb : MobileBanking("mobile_banking_scb")
```
Comment (nuxzero) [inline sdk/src/main/java/co/omise/android/models/SourceType.kt:38]: Not sure future plan, will we have more new MobileBankings in the future?

Comment (danfowler) [reply sdk/src/main/java/co/omise/android/models/SourceType.kt:38]: Yes, probably

**Comment by nuxzero** [comment]
@tookaa2 Could you add at least one QA to review this as well? 🙂

**Comment by nuxzero** [comment]
@tookaa2 Could you add some tests? 🙂

**Review by nuxzero — COMMENTED** [review]
Could you add the logo image to support the `mdpi` size?

File: sdk/src/main/java/co/omise/android/models/SourceType.kt (hunk 1 above)
Comment (tookaa2) [reply sdk/src/main/java/co/omise/android/models/SourceType.kt:38]: according to PO we will have BAY KBANK and more in future khub

**Comment by testyfruit** [comment]
QA:

- Mobile Banking SCB option available in settings in specifying method list (SCB - Mobile ) `qa:ok`
- New payment method added : Mobile Banking `qa:ok`
- Mobile Banking SCB added to payment list `qa:ok`
- Check that method is not displayed, when select payments 'from capabilities API', for the account, which didn't enable it `qa:ok`
- Check that method is displayed, when select payments 'from capabilities API', for the account, which support it `qa:ok`
- Verify source object structure `qa:ok`
- Create a charge with source object created by SDK `qa:ok`
- Create other payment methods object `qa:ok`

`qa:ok`

**Review by nuxzero — APPROVED** [review]
👍

**Pull request merged**